src/core/server/server.ts

```diff
@@ -212,12 +212,16 @@ export class Server {
 
     await this.http.start();
 
+    // Starting it after http because it relies on the the HTTP service
```

Comment:
You mean it relies on the `elasticsearch` service, right? Not `http`.

Reply:
It depends on ElasticSearch but it also depends on HTTP: we are making requests to the Kibana HTTP service to retrieve the instructions (at the moment, in the future it will be to Pulse)

Reply:
Got it